lib/ClangImporter/SwiftDeclSynthesizer.cpp

```diff
@@ -2489,24 +2489,23 @@ synthesizeDefaultArgumentBody(AbstractFunctionDecl *afd, void *context) {
   return {body, /*isTypeChecked=*/true};
 }
 
-CallExpr *
-SwiftDeclSynthesizer::makeDefaultArgument(const clang::ParmVarDecl *param,
-                                          const swift::Type &swiftParamTy,
-                                          SourceLoc paramLoc) {
+std::tuple<FuncDecl *, CallExpr *> SwiftDeclSynthesizer::makeDefaultArgument(
```

Comment:
Since this only has two elements, could it be a `pair` instead of `tuple`?

Reply:
Oh, yeah it could! I'll adjust that.